sql/_23_apricot_qa/_01_sql_extension3/_03_pseudocolumn_in_default_clause/_06_adhoc/cases/_01_adhoc_multiple.sql

```diff
@@ -613,11 +613,11 @@ CALL add_user('test2','test2') ON CLASS db_user;
 GRANT INSERT ON ALL t1 TO test1, test2;
 GRANT SELECT ON ALL t1 TO test1, test2;
 CALL login('test1','test1') ON CLASS db_user;
-insert into t1(b) values (1);
+insert into dba.t1(b) values (1);
 CALL login('test2','test2') ON CLASS db_user;
-insert into t1(b) values (2);
+insert into dba.t1(b) values (2);
 CALL login('dba','') ON CLASS db_user;
-select * from t1 order by 1,2;
+select * from dba.t1 order by 1,2;
```

Comment:
This line is after logging in as dba.
Since the t1 table is owned by dba, there is no need to specify the owner.
```
select * from t1 order by 1,2;
````

Reply:
Reflected the review. Please check again.